tokamak.py

```
@@ -0,0 +1,72 @@
+import argparse
+import cubit
+import yaml
+import numpy as np
+
+def parse_args():
+
+	parser = argparse.ArgumentParser(description='Create a tokamak geometry based on a radial build.')
+	parser.add_argument('radial_build', type=open, help='YAML file path describing tokamak radial build')
+	parser.add_argument('major_radius', type=float, help='plasma major radius [cm]')
+	parser.add_argument('minor_radius', type=float, help='plasma minor radius [cm]')
+	parser.add_argument('export_path', nargs='?', default='tokamak', help='path to export .sat file')
+	args = parser.parse_args()
+	return args
+
+def read_radial_build(file):
+	
+	radial_build = yaml.load(file, Loader=yaml.FullLoader)
+	n = list(radial_build.keys())
+	t = list(radial_build.values())
+	return n, t
+
+
+def build_torus(R,r,n,t):
+
+	T = 0.0
+	for i in t:
+		T += i
+	if T+r > R:
+		print("Error: inboard thickness plus plasma minor radius is greater than plasma major radius.")
```

Generally, better to throw an exception or return some error code. (separation of concerns)

---

```suggestion
		raise(ValueError, "inboard thickness plus plasma minor radius is greater than plasma major radius.")
```